src/test/suite/fixtures/recorded/languages/typescript/clearTypeOx.yml

```diff
@@ -0,0 +1,27 @@
+languageId: typescript
```

Comment:
This test case fails, but I believe it should pass

Reply:
I agree. Updated